samples/client/petstore/python-asyncio/petstore_api/configuration.py

```diff
@@ -35,7 +35,7 @@ class Configuration(object):
 
     def __init__(self, host="http://petstore.swagger.io:80/v2",
                  api_key=None, api_key_prefix=None,
-                 username="", password=""):
+                 username=None, password=None):
```

Comment:
If you make these default to None, then invoking 
config. get_basic_auth_token() will raise
`TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'`